Alicerce.xcodeproj/project.pbxproj

```diff
@@ -98,7 +100,9 @@
 			isa = PBXGroup;
 			children = (
 				1BDD79411E03840F00CC1EDA /* Extensions */,
+				1BE256D21E928FFB00942DAC /* View */,
```

Comment:
This duplicated reference seems odd. It must have slipped thru during conflict resolution 😝